app/views/search/index.html.slim

```diff
@@ -9,10 +9,14 @@
           .visible-xs
             = render template: "search/_mobile_categories.html.slim"
       .col-xs-12.col-md-8.col-lg-9.search.search-result
-        - case params[:type]
-          - when 'repositories'
-          = render template: "search/_repositories.html.slim"
-          -when 'namespaces'
-          = render template: "search/_namespaces.html.slim"
-          -when 'teams'
-          = render template: "search/_teams.html.slim"
+        .search-filter
```

Comment:
+1 for a helper method to DRY out the code. The hidden field can be set via a parameter - something like:
(psuedo code)
```ruby
def form_helper_name(with_hidden = false)
  form_tag that_creates_hidden_field if with_hidden
end
```

(just my 2 cents :smiley: )